packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -2,6 +2,16 @@ import asyncStates from '@zooniverse/async-states'
 import { autorun } from 'mobx'
 import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
 
+const NO_IMAGE_TOOLBAR_MIME_TYPES = [
+  'audio/mp3',
+  'audio/m4a',
+  'audio/mpeg',
+  'text/plain',
+  'video/mp4',
+  'video/mpeg',
+  'video/x-m4v'
```

Comment:
You can draw on videos, but I think this removes the ‘Annotate’ button from videos. 

Do we still plan to add the ‘Fullscreen' button, and corresponding layout, for images and videos?


If so, the logic changes here.

Reply:
Text files can be annotated too, with the highlighter task, but I don’t know if the ‘Annotate’ button is needed as an indicator that marking is enabled.

Reply:
A single Annotate button, that does nothing, would be a confusing experience but a layout that shifts when you switch between image and text would also be confusing UX. 

Better UX for image and text might be to keep the same layout for each frame, but disable elements that aren’t relevant to the active frame.